pkg/collector/runner/runner.go

```diff
@@ -258,7 +258,7 @@ func (r *Runner) work() {
 		doLog, lastLog := shouldLog(check.ID())
 
 		if doLog {
-			log.Infof("Running check %s", check)
+			log.Infoc("Running check", "check", check)
 		} else {
 			log.Debugf("Running check %s", check)
```

Comment:
If we're going to do this, let's make this also `log.Debugc()`, for consistency?

Reply:
You're right, done!